misk/src/main/kotlin/misk/web/interceptors/RebalancingInterceptor.kt

```diff
@@ -15,12 +16,19 @@ private val logger = getLogger<RebalancingInterceptor>()
  * When we're deploying and redeploying our pods, we want to make sure that clients rebalance onto
  * the new pods. This randomly closes connections so they will be recreated, naturally balancing
  * connections across pods.
+ *
+ * This does not close Unix domain socket connections.
```

Comment:
can we describe why here?

Reply:
it feels like a skim-type of thing to configure TBH, but seems reasonable to start with it here